randovania/games/dread/json_data/Artaria.txt

```diff
@@ -1134,6 +1140,8 @@ Extra - asset_id: collision_camera_010
   * Extra - left_shield_def: None
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
+  > Door to EMMI Zone Exit Northwest
+      Speed Booster
```

Comment:
This connection is redundant since you have speed connecting Exit Southwest and Exit Northwest. It is trivial to get to Southwest from here.